### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by robbiet480**
- Duration summary
- Multi-queue support

Thanks @mjgp2!

### CODE REVIEW

**Comment by pbadenski** [comment]
I'm a bit out of touch with changes in bull, could you explain a bit about this pull request?

**Comment by mjgp2** [comment]
Not bull specific, just adds the ability to bind to multiple queues, and a duration metric.

File: lib/index.js [hunk 1]
```diff
@@ -2,20 +2,29 @@
...
 function init(opts) {
-    const { queue, interval = 60000, promClient = client } = opts;
+    const { interval = 60000, promClient = client } = opts;
     const QUEUE_NAME_LABEL = 'queue_name';
     const activeMetricName = 'jobs_active_total';
     const waitingMetricName = 'jobs_waiting_total';
     const completedMetricName = 'jobs_completed_total';
     const failedMetricName = 'jobs_failed_total';
     const delayedMetricName = 'jobs_delayed_total';
+    const durationMetricName = 'jobs_duration';
```
Comment (pbadenski) [inline lib/index.js:12]: Could you change metric name to include time unit (milliseconds?). See: https://prometheus.io/docs/practices/naming/#metric-names

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "bull-prom",
-  "version": "1.1.0",
+  "version": "1.2.0",
```
Comment (pbadenski) [inline package.json:3]: Can you remove this change? I'm going to version and release separately.

**Review by pbadenski — CHANGES_REQUESTED** [review]

**Comment by robbiet480** [comment]
Closing in favor of #5

**Pull request closed without merging**